.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+   "cSpell.words": [
+      "unnesseccary"
```

Comment:
Das sollte nicht gemutet werden :D

Reply:
Ich wusste gar nicht das in den settings was geändert wurde 😅